.github/workflows/docker.build.yml

```diff
@@ -56,4 +56,24 @@ jobs:
           cache-to: type=local,mode=max,dest=/tmp/.buildx-cache
           push: ${{ github.event_name != 'pull_request' }}
           tags: ${{ steps.docker_meta.outputs.tags }}
-          labels: ${{ steps.docker_meta.outputs.labels }}
\ No newline at end of file
+          labels: ${{ steps.docker_meta.outputs.labels }}
+      #Create PR on deploy repo from a merged PR
+      -
+        name: create pull request for deploy on staging
+        uses: actions/checkout@v3
+        if: github.event.review.state == 'approved'
+        with:
+          repository: wmde/wbaas-deploy
+      # Make changes to pull request here
+      - run: |
+          get_tag=$(echo ${{ steps.docker_meta.outputs.tags }} | cut -d':' -f2)
+          yq -i '.image.tag=$get_tag' k8s/helmfile/env/local/ui.values.yaml.gotmpl
+          yq -i '.image.tag=$get_tag' k8s/helmfile/env/staging/ui.values.yaml.gotmpl
+        if: github.event.review.state == 'approved'
+      - uses: peter-evans/create-pull-request@v4
+        if: github.event.review.state == 'approved'
+        with:
+          commit-message: Staging+Local:Deploy new UI image ${{ steps.docker_meta.outputs.tags }}
+          title: 'Staging+Local: Deploy new UI image ${{ steps.docker_meta.outputs.tags }}'
+          labels: automated pr
+          token: ${{ secrets.GH_TOKEN_PLUSW }}
```

Comment:
Adam Shortland is the owner of the token and I think PLUSW is "plus write"

Reply:
If I understood the conversation in Mattermost correctly, we might want to use this user instead https://github.com/wmdebot who is managed by Leszek. This would make it easier to distinguish bot things from Adam things.

I think Conny (and all other EMs) should have the privileges required to create an access token for us. Should we ask her if she can help us?